composer.json

```diff
@@ -16,7 +16,7 @@
         "drupal/helfi_azure_fs": "^2.0",
         "drupal/helfi_drupal_tools": "dev-main",
         "drupal/helfi_navigation": "^2.0",
-        "drupal/helfi_platform_config": "^4.0",
+        "drupal/helfi_platform_config": "^4.2",
```

Comment:
Revert changes from this file. We don't update `helfi_*` modules or usually any other modules in Drupal instance's composer.json. Updates are done automatically by Github workflows when a new version of the module is released.

Reply:
Removed